InventorySales/etc/di.xml

```diff
@@ -141,9 +141,6 @@
         <plugin name="process_back_item_qty" type="Magento\InventorySales\Plugin\CatalogInventory\StockManagement\ProcessBackItemQtyPlugin"/>
         <plugin name="process_revert_products_sale" type="Magento\InventorySales\Plugin\CatalogInventory\StockManagement\ProcessRevertProductsSalePlugin"/>
     </type>
-    <type name="Magento\CatalogInventory\Api\RegisterProductSaleInterface">
-        <plugin name="process_register_products_sale" type="Magento\InventorySales\Plugin\CatalogInventory\StockManagement\ProcessRegisterProductsSalePlugin"/>
-    </type>
```

Comment:
just make sure that this plugin is no needed, and function Magento\CatalogInventory\Api\RegisterProductSaleInterface can't be executed out of order placement process. 
Because RegisterProductSaleInterface is an API which stays in the codebase

Reply:
Plugin restored